frontend/src/lib/StreamlitLib.test.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react"
+import {
+  Text as TextProto,
+  Block as BlockProto,
+  Element as ElementProto,
+  ForwardMsgMetadata as ForwardMsgMetadataProto,
+} from "src/lib/proto"
+import { BlockNode, ElementNode } from "./AppNode"
+import VerticalBlock from "./components/core/Block"
+import Text from "src/lib/components/elements/Text"
+import { ComponentRegistry } from "./components/widgets/CustomComponent"
+import { FileUploadClient } from "./FileUploadClient"
+import { mockEndpoints, mockSessionInfo } from "./mocks/mocks"
+import { ScriptRunState } from "./ScriptRunState"
+import { mount } from "./test_util"
+import { createFormsData, WidgetStateManager } from "./WidgetStateManager"
+
+function createMockBlockNode(scriptRunID: string, text: string): BlockNode {
+  const textElement = new ElementNode(
+    ElementProto.create({
+      text: TextProto.create({ body: text }),
+    }),
+    ForwardMsgMetadataProto.create(), // This doesn't actually matter
+    scriptRunID
+  )
+
+  return new BlockNode(
+    [textElement],
+    BlockProto.create({
+      vertical: BlockProto.Vertical.create({}),
+    }),
+    scriptRunID
+  )
+}
+
+describe("StreamlitLib", () => {
+  it("can be mounted", () => {
```

Comment:
This is a test that exists primarily to demonstrate how StreamlitLib can be used. (It's not testing new functionality; the reason it's implemented as a test is to guarantee it doesn't get out of date.)